Doc/tutorial/controlflow.rst

```diff
@@ -139,17 +139,16 @@ but in fact it isn't. It is an object which returns the successive items of
 the desired sequence when you iterate over it, but it doesn't really make
 the list, thus saving space.
 
-We say such an object is *iterable*, that is, suitable as a target for
+We say such an object is :term:`iterable`, that is, suitable as a target for
 functions and constructs that expect something from which they can
 obtain successive items until the supply is exhausted. We have seen that
-the :keyword:`for` statement is such an *iterator*. The function :func:`list`
-is another; it creates lists from iterables::
+the :keyword:`for` statement is such a construct, while an example of function
+that takes an iterable is :func:`sum`:
```

Comment:
Actually I haven't said anything about this particular example. Perhaps you meant to post this on the other PR?

Anyway, the tutorial covers the core features of the language so I don't think they are going to change a lot. Plus, I'm not fan of code churn and doctest usually makes examples less readable (especially if you read them in raw reST files which I do lot when doing code reviews)

Use of `sphinx-build` seems like a personal choice. People can have different workflows. For example, I always run my example code in REPL first and paste the output from REPL.